### ISSUE TITLE
Meeting setup information for funded DCC Onboarding

### ISSUE THREAD

**Issue by ACharbonneau** [nih-cfde/training-and-engagement#195]
1 hour, Zoom meeting per DCC
Invite @ACharbonneau, @owhite, @RLC-DCPPC, and the PIs from the DCC being onboarded. 

Do @kinsingc, @hresat and/or @kutkatl want to be invited to these? Please respond in a comment. If you only want to come for certain DCCs, please specify

@josesanchez1815 will be coordinating. These meetings need to start asap after the DCC is funded, so you should start finding times with the DCC as soon as I get you names to email for each one. Each DCC will get its own issue with contact info 

For each onboarding DCC, you (Jose) need to 

Before the meeting:
- [setup the spaces](https://github.com/nih-cfde/training-and-engagement/issues/196) for that DCC
- contact all the people who will be attending and set up a time they can do it (so that involves setting up a poll and choosing a time)
- pester them if necessary to get them to fill out the onboarding form
- make the zoom meeting
- send out the meeting invitation 
  - include the personalized agenda in the invitation
- make sure that everyone who has filled out the form is onboarded before the meeting

During the meeting:
- take notes and save them to their agenda
- record who needs to attend the biweekly PM meeting for that DCC, who will attend the Data Ingest meeting, and who will attend PM homeroom

After the meeting:
- Either find set up those new meetings, or add the appropriate people to already scheduled meetings
- Send any follow up information that they need
- Ensure everyone from their team gets onboarded if they didn't already

 @ACharbonneau : Please invite us to all DCC sessions. We can attend as our time allows. 

DCCs to start with are:  KidsFirst, GTEx, LINCS, SPARC, Metabolomics, ExRNA, IDG, and HuBMAP. There are 8 of them. Awards will go out very soon. I would not wait for the awards to be issued and contact their PIs and let them know about your plans asap. If you do not have the contact info for the PIs, let us know.

You can ask PIs to send the names of the folks that they want to participate from their groups. Being online and no-cost, you can tell them to be as inclusive as they want. This way decision and responsibility is with the PIs. They won't have any right to complain afterwards about missing someone.

**Comment by ACharbonneau** [comment]
Agenda Template
-----------------

Date:
Time:
Zoom meeting link:
Attendees:

Link to DCC github repo:
Link to this agenda:

- Introductions (all)
- Onboarding instructions review/help (Amanda)
- Intro to CFDE spaces (Amanda)
   - [Organization repo](https://github.com/nih-cfde/organization)
   - [groups.io](https://cfde.groups.io/g/General/topics)
   - Slack
   - Google Drive
      - [Everyone](https://drive.google.com/drive/u/2/folders/0ALAyQumxL5IgUk9PVA)
      - [Admin](https://drive.google.com/drive/u/2/folders/0AIspdLMpflayUk9PVA)
- Mini Github tutorial (Amanda, as needed)
- Zenhub overview (Amanda)
- Discuss regular meeting schedule (Amanda & Bob)
   - CC-DCC bi-weekly checkin (~30 min)
   - Data Ingest Team (~60 min)
   - PM Homeroom (~60 min)
   - RFC and workgroups (varies)
- Workplan discussion (Owen)

**Comment by ACharbonneau** [comment]
@josesanchez1815 Please come up with the template text for the email you'll send DCCs in this thread, and I'll help you edit it. That way all the templates are here. 

Your email should:
- be pretty short and to the point: you want to find times for their CFDE-CC Onboarding meeting. The meeting will give them an overview of our project management and reporting structure so they can get their deliverables into github
- include a link to the onboarding form and tell them we'd like them to onboard at least 2 business days before the meeting
- a link to a way for them to choose meeting times

**Comment by josesanchez1815** [comment]
# Draft E-mail for the DCCs

Greetings from the Common Fund Data Ecosytem Coordinating Center!

We would like to invite you to join us at the CFDE-CC. We look forward to start collaborating with you! You will be playing a vital role in the development of CFDE as we strive to connect the NIH Common Fund Programs together to discover solutions to new and existing medical challenges.

In order for you to be onboarded, we need to get you set-up with the resources you will need.  We would like to set-up a meeting with you to go over project management and communication resources available to us as well as discuss what our plans are moving forward. Please let us know what your availability in this doodle poll insert doodle poll, so that we coordinate a zoom meeting. Prior to this meeting, we would like to set up your accounts for Slack, Google Drive, and our CFDE Training Repo. If you could fill out our [onboarding form](https://docs.google.com/forms/d/e/1FAIpQLSeMEmXWv63Fa2FpV1ssQ-xxiux0-dhx5OBnj2SHhFVlXBbpTQ/viewform) at least two days prior to our scheduled meeting, we could prepare your resources and discuss them during our zoom meeting.

Attached to this e-mail, you will find an attached agenda for our Onboarding Meeting. 

Please let us know if you have any questions or concerns.

Best,



Jose
Training Coordinator
Common Fund Data Ecosystem Coordinating Center
****************************************************************************
Rough Draft 2
Greetings from the Common Fund Data Ecosystem Coordinating Center!

We would like to invite you to join us at the CFDE-CC. We look forward to collaborating with you! You will be playing a vital role in the development of CFDE as we strive to connect the NIH Common Fund Programs together to discover solutions to new and existing medical challenges.

We would like to set-up a meeting with you to go over 
- project management and communication resources  
- discuss our plans for the CFDE 2021

Could you please let us know your availability in this doodle link, so that we may schedule a zoom meeting accordingly?

Prior to this meeting, we would like to set up your accounts for Slack, Google Drive, and our CFDE Training Repo. If you could fill out our [onboarding form](https://docs.google.com/forms/d/e/1FAIpQLSeMEmXWv63Fa2FpV1ssQ-xxiux0-dhx5OBnj2SHhFVlXBbpTQ/alreadyresponded) at least two days prior to our scheduled meeting, we could prepare and discuss them during our zoom meeting.

Attached to this e-mail, you will find an attached agenda for our Onboarding Meeting. 

Please let us know if you have any questions or concerns.

Best,


Jose
Training Coordinator
Common Fund Data Ecosystem Coordinating Center

**Comment by hresat** [comment]
> 1 hour, Zoom meeting per DCC
> Invite @ACharbonneau, @owhite, @RLC-DCPPC, and the PIs from the DCC being onboarded.
> 
> Do @kinsingc, @hresat and/or @kutkatl want to be invited to these? Please respond in a comment. If you only want to come for certain DCCs, please specify
> 
> @josesanchez1815 will be coordinating. These meetings need to start asap after the DCC is funded, so you should start finding times with the DCC as soon as I get you names to email for each one. Each DCC will get its own issue with contact info
> 
> For each onboarding DCC, you (Jose) need to
> 
> Before the meeting:
> 
> * [setup the spaces](https://github.com/nih-cfde/training-and-engagement/issues/196) for that DCC
> * contact all the people who will be attending and set up a time they can do it (so that involves setting up a poll and choosing a time)
> * pester them if necessary to get them to fill out the onboarding form
> * make the zoom meeting
> * send out the meeting invitation
>   
>   * include the personalized agenda in the invitation
> * make sure that everyone who has filled out the form is onboarded before the meeting
> 
> During the meeting:
> 
> * take notes and save them to their agenda
> * record who needs to attend the biweekly PM meeting for that DCC, who will attend the Data Ingest meeting, and who will attend PM homeroom
> 
> After the meeting:
> 
> * Either find set up those new meetings, or add the appropriate people to already scheduled meetings
> * Send any follow up information that they need
> * Ensure everyone from their team gets onboarded if they didn't already

**Comment by hresat** [comment]
Sorry - clicked on the wrong thing :( 

@ACharbonneau : Please invite us to all DCC sessions. We can attend as our time allows.

DCCs to start with are: KidsFirst, GTEx, LINCS, SPARC, Metabolomics, ExRNA, IDG, and HuBMAP. There are 8 of them. Awards will go out very soon. I would not wait for the awards to be issued and contact their PIs and let them know about your plans asap. If you do not have the contact info for the PIs, let us know.

You can ask PIs to send the names of the folks that they want to participate from their groups. Being online and no-cost, you can tell them to be as inclusive as they want. This way decision and responsibility is with the PIs. They won't have any right to complain afterwards about missing someone.

**Comment by kinsingc** [comment]
Please invite kinsingc to all DCC engagement meetings.  Please require me to attend meetings with Kids First, GTEx, and Metabolomics.

**Comment by kinsingc** [comment]
I'm making some corrections to the text:

Greetings from the Common Fund Data Ecosytem Coordinating Center (CFDE-CC)!

We would like to invite you to join us at the CFDE-CC. We look forward to collaborating with you! You will play a vital role in the development of the CFDE as we strive to connect the NIH Common Fund Programs to discover solutions to new and existing medical challenges.

In order for to onboard you, we need to orient you with the resources you will need. We would like to set up a meeting with you to cover CFDE project management and communication resources, as well as discuss our plans for moving forward. Please let us know your availability in this doodle poll [_insert doodle poll_], so that we can coordinate a Zoom meeting. Prior to this meeting, we would like to set up your accounts for Slack, Google Drive, and our CFDE Training Repo. If you could fill out our onboarding form at least two days prior to our scheduled meeting, we will prepare your resources and discuss them during our Zoom meeting.

Attached to this e-mail, you will find an attached agenda for our Onboarding Meeting.

Please let us know if you have any questions or concerns.

Best,

Jose Sanchez
Training Coordinator
Common Fund Data Ecosystem Coordinating Center

**Comment by jeremywalter** [comment]
Reduced version -- in markdown -- check format if sending:  

Welcome to the Common Fund Data Ecosystem!

In anticipation of your funding announcement, we're having a [insert DCC here] onboarding meeting to get your management team oriented to our project management and reporting structure. You can help us prepare you for the meeting by:

- [ ] filling out this doodle poll [insert doodle poll] so that we can coordinate a Zoom meeting.

- [ ] forwarding this message to anyone else on your team who should attend the meeting- please cc ronsanchez@ucdavis.edu.

- [ ] filling out our [onboarding form](https://docs.google.com/forms/d/e/1FAIpQLSeMEmXWv63Fa2FpV1ssQ-xxiux0-dhx5OBnj2SHhFVlXBbpTQ/viewform) at least 2 business days before the meeting so that we can set up relevant accounts necessary for the meeting.

We need each person from [specific DCC]who will be interacting with the project management system to be onboarded, so they would each fill out the onboarding form. If you onboarded previously, you don’t need to do it again.

Attached you will find an agenda for our Onboarding Meeting.

Please let us know if you have any questions or concerns.

Best,


Jose Sanchez
Training Coordinator
Common Fund Data Ecosystem Coordinating Center

**Comment by josesanchez1815** [comment]
GTEx https://doodle.com/poll/5uivrym65db5kkhk
doodle link

**Comment by ACharbonneau** [comment]
@josesanchez1815 can you put the individual info for each call on their individual issues? Otherwise this issue is going to get super hard to follow. So put GTEx here: https://github.com/nih-cfde/training-and-engagement/issues/208 thanks

That way you can put the exact email text in the issue and everything :)

**Comment by ACharbonneau** [comment]
@josesanchez1815 can you add @ctb to meeting emails and doodle polls from now on, and also forward him the ones we've already done in case he can attend?

**Comment by josesanchez1815** [comment]
Funded DCC Onboarding Completed!
